coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/DatabaseAccessors.scala

```diff
@@ -372,10 +378,31 @@ object DatasetMutator {
             }
           }
           copyCtx = new DatasetCopyContext(newCi, datasetMap.schema(newCi)).thaw()
+          dropInvalidRollups(newCi)
           copyInfo
         }
       }
 
+      private def dropInvalidRollups(copyInfo: CopyInfo): Unit = {
+        val prefixedSchema: Seq[(ColumnName, CT)] =
+          datasetMap.schema(copyInfo).values.map { colInfo =>
+            (new ColumnName("_" + colInfo.userColumnId.underlying), colInfo.typ)
+          }(collection.breakOut)
+        val prefixedDsContext = new com.socrata.soql.environment.DatasetContext[CT] {
+            val schema: OrderedMap[ColumnName, CT] = OrderedMap(prefixedSchema: _*)
+        }
+
+        datasetMap.rollups(copyInfo).foreach { (ru: RollupInfo) =>
+          try {
+            soqlAnalyzer.analyzeFullQuery(ru.soql)(prefixedDsContext)
```

Comment:
This will re-evaluate `soqlAnalyzer` for each rollup.  Probably not a big deal but just making sure this is deliberate..